Source/Document Structure/SvgScript.cs

```diff
@@ -0,0 +1,81 @@
+using System.Drawing;
+using System.Drawing.Drawing2D;
+
+namespace Svg
+{
+    /// <summary>
+    /// An element used to define scripts within SVG documents.
+    /// Use the Script property to get the script content (proxies the content)
+    /// </summary>
+    [SvgElement("script")]
+    public class SvgScript : SvgElement
+    {
+
+        public string Script 
+        {
+            get { return this.Content; }
+            set { this.Content = value; }
+        }
+
+        private string _scriptType;
+
+        [SvgAttribute("type")]
+        public string ScriptType 
+        { 
+            get { return _scriptType; } 
+            set { _scriptType = value; } 
+        }
+
+        private string _crossOrigin;
+
+        [SvgAttribute("crossorigin")]
+        public string CrossOrigin 
+        {
+            get { return _crossOrigin; }
+            set { _crossOrigin = value; }
+        }
+
+        private string _href;
+        [SvgAttribute("href")]
+        public string Href
+        {
+            get { return _href; }
+            set { _href = value; }
+        }
+    
+        public override SvgElement DeepCopy()
+        {
+            return DeepCopy<SvgScript>();        
+        }
+
+        public override bool ShouldWriteElement() 
```

Comment:
I think that it is not necessary because there is `SvgElementAttribute`.

Reply:
I did not test the behaviour without the override, but I did have some troubles with the XML generation with regard to the attributes not being written, so I tinkered around with the code. The ShouldeWriteElement indeed is not necessary, so I dropped it in the new commit.